### ISSUE TITLE
[Issue] --cluster flag builds concurrently causing dependencies to fail

### ISSUE THREAD

**Issue by 8ball030** [valory-xyz/open-autonomy#1154]
# The --dependencies flag builds in parrallel. This means i am unable to build the dependency images in the the cluster.

This is a build using the --cluster flag. Which fails and does not appear to return a status code 1

![image](https://user-images.githubusercontent.com/35799987/182327052-87cad6a2-d0ae-4f3f-9c7b-71e7d0602ef9.png)

Here is a build which *passes* in the cluster but only of the oracle service images.

```

This shows us a few things;

We are able to connect to the docker daemon to login
we are failing to build the images, YET we are still exiting with a status code 0
we are building in parallel on the.
I believe that we are unable to build the dependencies due to issue number 3.
Yet, i believe that the fact that my initial work flow to build the service works perfectly we have an issue with the library.

that can be viewed as 2 problems;
1 We dont seem to throw an exit code on failure for this build. I can back this up with logs.
2. we build in parallel on the --cluster profile.

Moving forwards;

If i can get a branch with skaffold invoked with build--concurrency=1 for the --cluster profile that would be super appreciated. we can confirm if this is an issue for the kaniko runner.

I could invoke skaffold

I can run a number of different varients to disprove my hypothesis of the incorrect exit code.

A review of my invocation commands from @angrybayblade would be appreciated.

```

### Your environment
- Python version: [e.g. 3.7.2]
- Package Version [e.g. 0.1.3]
- Anything else you consider helpful.

### Steps to reproduce
Tell us how to reproduce this issue.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

**Comment by DavidMinarsch** [comment]
No longer relevant since `0.2.1`
